src/spawn.cpp

```diff
@@ -33,7 +33,8 @@ extern Monsters g_monsters;
 extern Game g_game;
 extern Events* g_events;
 
-static constexpr int32_t MINSPAWN_INTERVAL = 1000;
+static constexpr int32_t MINSPAWN_INTERVAL = 10000; // 10 seconds to match RME
+static constexpr int32_t MAXSPAWN_INTERVAL = 86400000; // 1 day
```

Comment:
What about using `std::chrono::seconds` here?

Reply:
Can you elaborate?
found some interesting docs though:
https://en.cppreference.com/w/cpp/chrono/duration
chrono durations might be a better way generally to deal interval and timed stuff

Do you want the type declaration to be changed to std::chrono::seconds? Or do you want this entire thing to be chrono-fyed?

Reply:
I suggest the whole thing to use chrono::seconds.